packages/common/src/core/types/IHubInitiative.ts

```diff
@@ -15,4 +11,13 @@ export interface IHubInitiative
   extends IHubItemEntity,
     IWithSlug,
     IWithCatalog,
-    IWithPermissions {}
+    IWithPermissions {
+  /**
+   * Definition of any dynamic values that can be resolved
+   */
+  dynamicValues?: DynamicValueDefinition[];
+  /**
+   * Container for values that can be the source for other dynamic values
+   */
+  values?: DynamicValues;
```

Comment:
I don't understand this

Reply:
This is one of the things that's hard to name - it's the prop, on the entity, which holds the dynamic values. In the Initiative/Project scenario, this would be `myProject.values.funding`.